src/subclass/object.rs

```diff
@@ -659,7 +660,7 @@ mod test {
                 .expect("Failed to get str from args[1]");
 
             assert_eq!(name, "new-name");
-            *name_changed_clone.lock().expect("Failed to lock") = true;
+            name_changed_clone.store(true, Ordering::Relaxed);
```

Comment:
Is relaxed ordering enough here (and elsewhere)? `SeqCst` would be the safe choice.

Reply:
This is a basic test and not actually using parallelism. As far as I can tell the use of `Mutex<bool>` could just as easily have been replaced with `RefCell<bool>` as `AtomicBool`, I just went with the default conversion I've used in test suites in the past.  Your call.

Reply:
My rule is usually that it's wrong unless someone can explain to me why anything else than `SeqCst` is correct :) You're right here with the test of course, so let's leave it.

Reply:
I use `Relaxed` when I know it's safe (e.g in these tests) and then whenever I see `Relaxed` I know to be aware that if I change things I may need to reassess.  I'll leave this as-is.